src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectReportController.java

```diff
@@ -93,12 +95,55 @@ public void getReport(HttpServletResponse response,
 
         byte[] content = catalogObjectReportService.generateBytesReport(authorisedBucketsNames, kind, contentType);
 
+        flushResponse(response, content);
+
+    }
+
+    @ApiOperation(value = "Get list of selected catalog objects in a PDF report file")
+    @ApiResponses(value = { @ApiResponse(code = 404, message = "Bucket not found"),
+                            @ApiResponse(code = 401, message = "User not authenticated"),
+                            @ApiResponse(code = 403, message = "Permission denied") })
+    @RequestMapping(value = "/selected/{bucketName}", method = GET)
+    @ResponseStatus(HttpStatus.OK)
+    public void getReportForSelectedObjects(HttpServletResponse response,
+            @ApiParam(value = "sessionID") @RequestHeader(value = "sessionID", required = false) String sessionId,
+            @ApiParam(value = "The name of the user who owns the Bucket") @RequestParam(value = "owner", required = false) String ownerName,
+            @PathVariable String bucketName,
+            @ApiParam(value = "Filter according to kind.") @RequestParam(required = false) Optional<String> kind,
+            @ApiParam(value = "Filter according to content type.") @RequestParam(required = false) Optional<String> contentType,
+            @ApiParam(value = "Give a list of name separated by comma to get them in the report", allowMultiple = true, type = "string") @RequestParam(value = "name", required = false) Optional<List<String>> catalogObjectsNames)
+            throws NotAuthenticatedException, AccessDeniedException, IOException {
+
+        if (sessionIdRequired) {
+            restApiAccessService.checkAccessBySessionIdForBucketAndThrowIfDeclined(sessionId, bucketName);
+        }
+
+        byte[] content = new byte[0];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "content". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)